azkaban-common/src/main/java/azkaban/executor/FetchActiveFlowDao.java

```diff
@@ -266,12 +268,12 @@ public FetchActiveFlowDao(final DatabaseOperator dbOperator) {
 
     // Select the flow that is dispatched and not in finished status by execution id
     private static final String FETCH_ACTIVE_EXECUTABLE_FLOW_BY_EXEC_ID =
-        "SELECT ex.exec_id exec_id, ex.enc_type enc_type, ex.flow_data flow_data, et.host host, "
-            + "et.port port, ex.executor_id executorId, et.active executorStatus"
+        "SELECT ex.exec_id exec_id, ex.enc_type enc_type, ex.flow_data flow_data, ex.status status,"
```

Comment:
If possible, add new column in the end. That way, you won't need to worry about correcting indexes of rest of the fields.

Reply:
We're accessing the columns by name when processing the results of this query so it's safe to add the column in the middle. 
We should probably use names in the rest of the queries too.